version.gradle.kts

```diff
@@ -34,7 +34,7 @@
 /**
  * The version of this library.
  */
-val base = "1.6.7"
+val base = "1.6.8"
```

Comment:
Shouldn't the library version be updated? And shouldn't our CI catch that it's not updated? Please investigate why the build is "green" while the published version has not changed.

Reply:
The PR was created and before the `1.6.8` version was merged in the master. So I guess everything is fine here. I'll bump the version up.

![image](https://user-images.githubusercontent.com/23313616/98791447-c0a2c680-240d-11eb-98bf-6fa2ab4196d7.png)

Reply:
Sorry, my bad. I thought I was looking at the `core-java`, for some reason.